lib/minurl.browser.js

```diff
@@ -0,0 +1,70 @@
+/// <reference lib="dom" />
```

Comment:
The types aren’t loaded conditionally. This means that vfile now always pulls in the `dom` lib, which is undesirable.

Reply:
`minurl.browser.js` types aren’t exported, TS doesn’t understand the `browser` field in `package.json`, but bundlers do.
Hence, it doesn’t really matter what types are inside this file, except when running TS on this file

Reply:
Ah, sounds great then :+1: